modules/training/php/training.class.inc

```diff
@@ -0,0 +1,229 @@
+<?php
```

Comment:
shouldn't the old file be removed too?

Reply:
@driusan 
You are right, the old file should be removed. It is done.